### DISCUSSION TITLE
Placement of tooltip is not working if we click on back button

### DISCUSSION THREAD

**Discussion by chandra9154** [gilbarbara/react-joyride#800] [General]
Placement of tooltip is not working if we click on back button. placement is showing as auto if we click on back button
steps: [
        {
          content: <h6 style={{color:'#00BCFF'}}>Search your product here</h6>,
          target: '.tour-product-pack',
          disableBeacon: true,
          locale: { skip: <strong aria-label="skip" style={{color:'#FF3162'}}>SKIP</strong> },
          placement:"top" as const
        },]

**Comment by gilbarbara** [comment]
hey @chandra9154 

Your example code is broken and without a functioning minimal example, it is impossible to debug it.
